src/explore-education-statistics-admin/src/components/comments/AddCommentForm.tsx

```diff
@@ -0,0 +1,122 @@
+import styles from '@admin/components/comments/AddCommentForm.module.scss';
+import releaseContentCommentService, {
+  AddComment,
+} from '@admin/services/releaseContentCommentService';
+import { Comment } from '@admin/services/types/content';
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import ButtonText from '@common/components/ButtonText';
+import { Form } from '@common/components/form';
+import FormFieldTextArea from '@common/components/form/FormFieldTextArea';
+import Yup from '@common/validation/yup';
+import useMounted from '@common/hooks/useMounted';
+import { Formik } from 'formik';
+import React, { RefObject, useRef } from 'react';
+import useToggle from '@common/hooks/useToggle';
+import classNames from 'classnames';
+
+interface FormValues {
+  content: string;
+}
+
+interface Props {
+  blockId: string;
+  containerRef?: RefObject<HTMLDivElement>;
+  releaseId?: string;
+  sectionId?: string;
+  onCancel: () => void;
+  onSave: (comment: Comment) => void;
+}
+
+const AddCommentForm = ({
+  blockId,
+  containerRef,
+  releaseId,
+  sectionId,
+  onCancel,
+  onSave,
+}: Props) => {
+  const [isSubmitting, toggleSubmitting] = useToggle(false);
+  const [fixPosition, toggleFixPosition] = useToggle(false);
+  const [focus, toggleFocus] = useToggle(false);
```

Comment:
It's needed as don't want to focus the input until after the position is set. Otherwise you can get scrolled back up to the top of the block away from the text you're commenting on.